src/components/Timer.tsx

```diff
@@ -33,9 +34,67 @@ const H1 = styled.h1<TypographyProps & FlexboxProps>`
 const Span = styled.span<SpaceProps>`
   ${space}
 `;
+
+const launchDates: LaunchDateProps[] = [
+  {
+    date: "-10-2",
+    url: "/",
+  },
+  {
+    date: "-10-9",
+    url: "/conscious-shopping",
+  },
+  {
+    date: "-10-16",
+    url: "/marc-medina",
+  },
+  {
+    date: "-10-23",
+    url: "/erotic-stories",
+  },
+  {
+    date: "-10-30",
+    url: "/editorial",
+  },
+  {
+    date: "-11-6",
+    url: "/the-map",
+  },
+  {
+    date: "-11-13",
+    url: "/eyes",
+  },
+  {
+    date: "-11-20",
+    url: "/belledejour",
+  },
+  {
+    date: "-11-27",
+    url: "/leo-adef",
+  },
+  {
+    date: "-11-27",
+    url: "/kai-landre",
+  },
+];
+
+interface LaunchDateProps {
+  date: string;
+  url: string;
+}
+
+const GetDate = () => {
+  const location = useLocation();
+  launchDates.forEach((launchDate) => {
+    if (launchDate.url.includes(location.pathname)) {
+      return launchDate.date;
+    }
+  });
+};
+
 const calculateTimeLeft = () => {
   let year = new Date().getFullYear();
-  const difference = +new Date(`${year}-10-2`) - +new Date();
+  const difference = +new Date(`${year}${GetDate}`) - +new Date();
```

Comment:
Lovely yes, one thing -- here my preference would be to use the `Number` constructor to perform the type conversion, rather than the `+` operator, ie `Number(new Date())` because its effect is clearer.
@sssoja @Jbarget how do each of you feel about that?

Reply:
never seen this before! i'm down